Fsxac/proposal risk edits (#16)
Capstone-Skynet/Capstone-Skynet.github.io: docs/proposal/3_8_risk_profile.tex

The risk profile is updated/reviewed weekly (during the team meeting) and as otherwise necessary. 

\subsubsection{Potential Risks}

\begin{center}

% Make the table more stretched out
\renewcommand{\arraystretch}{1.3} % Default value: 1

\begin{longtable}{p{0.6\linewidth} *{3}{r}}

\caption{A risk profile table outlining the a list of potential risks.}\\
\hline
\textbf{Risk description} & \textbf{Likelihood} & \textbf{Impact} & \textbf{Risk ($\downarrow$)} \\
\hline
\endfirsthead

\multicolumn{4}{c}%
{\tablename\ \thetable\ -- \textit{Continued from previous page}} \\
\hline
\textbf{Risk description} & \textbf{Likelihood} & \textbf{Impact} & \textbf{Risk ($\downarrow$)} \\
\hline
\endhead

\hline \multicolumn{4}{r}{\textit{Continued on next page}} \\
\endfoot

\hline
\endlastfoot

> Drone flight hardware (flight controllers, radio, motors) cannot function to crashes and damages. & 0.9 & 1.0 & 0.90 \\ \hline

Reviewer: ```suggestion
Drone flight hardware (flight controllers, radio, motors) cannot function due to crash/damage. & 0.9 & 1.0 & 0.90 \\ \hline
```

Payload is too heavy which significantly increases drone motor requirements and significant reduction in flight duration. & 0.8 & 0.8 & 0.64 \\ \hline
Accidents that damage the drone and computating equipment that require extra budget that we may not have. & 0.6 & 0.9 & 0.54\\ \hline
Loss of drone hardware and payload during flight and cannot be recovered (total loss). & 0.5 & 1.0 & 0.50 \\\hline
Not enough time commitment from team members. & 0.7 & 0.7 & 0.49\\ \hline
Access to tools and shops for modifiying and repairing drone hardware is inadequate or non-existent. & 0.6 & 0.8 & 0.48\\ \hline

> Underestimation of project scope or work required, leading to insufficient time managemnet and burn-outs. & 0.5 & 0.9 & 0.45\\ \hline

Reviewer: ```suggestion
Underestimation of project scope or work required, leading to insufficient time management and burn-outs. & 0.5 & 0.9 & 0.45\\ \hline
```

Payload is too heavy which exceeds total take-off weight. & 0.4 & 1.0 & 0.40 \\ \hline

> Legacy documents for the project is insufficient, resulting in the final product unable to be maintained or extended. & 0.7 & 0.5 & 0.35\\ \hline

Reviewer: ```suggestion
Legacy documents for the project are insufficient, resulting in poor maintainability/extensibility for the client. & 0.7 & 0.5 & 0.35\\ \hline
```

Financial inefficiencies leading to overbudget or lack of capital. & 0.4 & 0.8 & 0.32\\ \hline

> Constrained to purchase lower-quality components due to budget and results in lower performance. & 0.6 & 0.5 & 0.30 \\ \hline

Reviewer: ```suggestion
Constrained to purchase lower-quality components due to budget, resulting in lower performance. & 0.6 & 0.5 & 0.30 \\ \hline
```

Team is indecisive or team leader cannot make a timely decision --- results in delay. & 0.4 & 0.6 & 0.30\\ \hline

> Development and managment technique and methodology is not effective, leading to productivity losses. & 0.4 & 0.7 & 0.28\\ \hline

Reviewer: ```suggestion
Development and management technique/methodology is not effective, leading to productivity losses. & 0.4 & 0.7 & 0.28\\ \hline
```

Author: What do you have against "and" or "or"s 😠

Not enough time to work on documentation. & 0.7 & 0.4 & 0.28\\ \hline

> Scope of the project is overestimated which leads to diversification of vision of the project. & 0.5 & 0.5 & 0.25\\ \hline

Reviewer: ```suggestion
Scope of the project is overestimated which leads to diversification of the project's vision. & 0.5 & 0.5 & 0.25\\ \hline
```

Not enough machine learning training data. & 0.5 & 0.5 & 0.25\\ \hline
Not enough FPGA logic elements to implement a desired ML model. & 0.5 & 0.5 & 0.25\\ \hline
Failure to acquire regulatory compliance resulting in inability fly drone legally. & 0.3 & 0.7 & 0.21\\ \hline
The software, tools or development environment for the project is inadequate. & 0.4 & 0.5 & 0.20\\ \hline
Knowledge and skill regarding ML is insufficient. & 0.5 & 0.4 & 0.20 \\ \hline

> Technical debt paydown impact project timeline. & 0.4 & 0.5 & 0.20\\ \hline

Reviewer: ```suggestion
Technical debt paydown impacts project timeline. & 0.4 & 0.5 & 0.20\\ \hline
```

> Deliverables and objectives failing to meeting client’s expectations. & 0.2 & 0.9 & 0.18 \\ \hline

Reviewer: ```suggestion
Deliverables and objectives fail to meet client’s expectations. & 0.2 & 0.9 & 0.18 \\ \hline
```

Client demands modification to the scope and requirements of the project that leads to delays or feature cuts. & 0.3 & 0.6 & 0.18\\ \hline
Team lacking in effective communication which leads to overlapping work, missed work, and incompatible work. & 0.4 & 0.4 & 0.16\\ \hline
Internal documentation or documentation for libraries and parts are not sufficient for development. & 0.4 & 0.4 & 0.16\\ \hline
Camera module lacking in documentation. & 0.2 & 0.8 & 0.16\\ \hline

> New technology or research that changes the scope significantly. & 0.2 & 0.5 & 0.10\\ \hline

Reviewer: ```suggestion
New technology or research emerges, changing the scope significantly. & 0.2 & 0.5 & 0.10\\ \hline
```

Sabotage of the project. & 0.1 & 1 & 0.10\\ \hline
Sudden loss of client. & $\leq$0.1 & 1 & 0.10\\ \hline
Sudden loss of team member. & $\leq$0.1 & 0.9 & 0.09\\ \hline
FPGA board lacks documentation. & $\leq$0.1 & 0.8 & 0.08\\ \hline
Client is not cooperative or does not provide necessary information. & $\leq$0.1 & 0.8 & 0.08\\ \hline
Key components are not available or takes too long to deliver. & 0.1 & 0.7 & 0.07\\ \hline

> Purchased orders of equipment or tools delayed or lost. & 0.1 & 0.5 & 0.05\\ \hline

Reviewer: Same as one directly above?

Author: Hmm, it appears that both were in there in the beginning but with different risk values, I rearranged and they're now together (which is a good thing I guess).

Client is not available enough to provide significant help. & $\leq$0.1 & 0.5 & 0.05\\ \hline

> Lacking resources in machine learning knowledge. & $\leq$0.1 & 0.5 & 0.05\\ \hline

Reviewer: Already listed?

Author: One is lacking the knowledge to do something, the other is lacking the resource to acquire the knowledge. I'll make an edit to make this clearer

> Camera module fail to interface with FPGA. & $\leq$0.1 & 0.5 & 0.05\\ \hline

Reviewer: ```suggestion
Camera module fails to interface with FPGA. & $\leq$0.1 & 0.5 & 0.05\\ \hline
```

> Data transmitter fail to interface with FPGA. & $\leq$0.1 & 0.5 & 0.05\\ \hline

Reviewer: ```suggestion
Data transmitter fails to interface with FPGA. & $\leq$0.1 & 0.5 & 0.05\\ \hline
```

> Competitions on the market significantly affecting project requirement and scope. & $\leq$0.1 & 0.4 & 0.04\\ \hline

Reviewer: ```suggestion
Market competition significantly affects project requirements and scope. & $\leq$0.1 & 0.4 & 0.04\\ \hline
```
